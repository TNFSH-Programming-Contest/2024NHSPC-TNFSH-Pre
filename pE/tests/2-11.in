15
954610 994299 973694 902542 903506 949879 917908 963702 900932 938585 901300 995966 908060 916004 940865 1
