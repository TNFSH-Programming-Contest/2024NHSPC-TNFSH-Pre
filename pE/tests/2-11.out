10958541455936